forge-gui/res/cardsfolder/upcoming/saheeli_filigree_master.txt

```diff
@@ -0,0 +1,14 @@
+Name:Saheeli, Filigree Master
+ManaCost:2 U R
+Types:Legendary Planeswalker Saheeli
+Loyalty:3
+A:AB$ Scry | Cost$ AddCounter<1/LOYALTY> | Planeswalker$ True | ScryNum$ 1 | SubAbility$ DBDraw | SpellDescription$ Scry 1. You may tap an untapped artifact you control. If you do, draw a card.
+SVar:DBDraw:DB$ Draw | UnlessCost$ tapXType<1/Artifact>
```

Comment:
`UnlessPayer$ You` and you also need UnlessSwitched since it's not a true Unless cost